packages/react-core/src/demos/DataListDemo.md

```diff
@@ -418,3 +426,346 @@ class ExpandableDataList extends React.Component {
   }
 }
 ```
+
+### Basic
+
+```js isFullscreen
+import React from 'react';
+import {
+  Button,
+  DataList,
+  DataListItem,
+  DataListCell,
+  DataListItemRow,
+  DataListItemCells,
+  Flex,
+  FlexItem,
+  MenuToggle,
+  MenuToggleCheckbox,
+  OverflowMenu,
+  OverflowMenuControl,
+  OverflowMenuItem,
+  PageSection,
+  PageSectionVariants,
+  Pagination,
+  Text,
+  TextContent,
+  TextVariants,
+  Toolbar,
+  ToolbarItem,
+  ToolbarContent
+} from '@patternfly/react-core';
+import EllipsisVIcon from '@patternfly/react-icons/dist/esm/icons/ellipsis-v-icon';
+import DashboardWrapper from '@patternfly/react-core/src/demos/examples/DashboardWrapper';
+
+import CodeBranchIcon from '@patternfly/react-icons/dist/esm/icons/code-branch-icon';
+import CodeIcon from '@patternfly/react-icons/dist/esm/icons/code-icon';
+import CubeIcon from '@patternfly/react-icons/dist/esm/icons/cube-icon';
+import ExclamationTriangleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
+import TimesCircleIcon from '@patternfly/react-icons/dist/esm/icons/times-circle-icon';
+
+class DataListViewBasic extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      page: 1,
+      totalItemCount: 5
+    };
+  }
+
+  renderPagination() {
+    const { page, totalItemCount } = this.state;
+
+    return <Pagination itemCount={totalItemCount} page={page} variant="top" isCompact />;
+  }
+
+  render() {
+    const toolbarItems = (
+      <React.Fragment>
+        <ToolbarItem variant="bulk-select">
+          <MenuToggle
+            aria-label="Select cards"
+            splitButtonOptions={{
+              items: [<MenuToggleCheckbox id="split-dropdown-checkbox" aria-label={'Select all cards'} />]
+            }}
+            key="split-dropdown-checkbox"
```

Comment:
This key should be moved up to the `MenuToggleCheckbox`, its being applied to `MenuToggle` atm.

Everything else lgtm